imagine/fields/base_fields.py

```diff
@@ -131,16 +132,22 @@ def data_description(self):
     def data_shape(self):
         return(None)
 
+    @property
+    def parameters_list(self):
+        """Parameters of the field"""
+        return [k for k in self.field_checklist]
+
     @property
     def field_checklist(self):
-        return({})
+        """Parameters of the dummy field"""
+        return getattr(self, 'FIELD_CHECKLIST', {})
```

Comment:
Isn't this (and `simulator_controllist`) a required attribute?

Reply:
It is a good question. It depends on the typical use-case of dummy fields. 

At first I thought  that we could keep the two (`field_checklist`/`simulator_controllist`) as optional, but maybe, for the sake of clarity, we should enforce them (the user can set them to an empty dict if needed).

I will amend this now.